src/plugins/intel_cpu/tests/functional/shared_tests_instances/snippets/fake_quantize_decomposition_test.cpp

```diff
@@ -85,7 +85,7 @@ INSTANTIATE_TEST_SUITE_P(
         ::testing::ValuesIn(testValuesDecompositionPerChannel),
         ::testing::ValuesIn(operations),
         // reorder (nChw[16|8]c) + MaxPool + reorder(nChw[16|8]c) x6 + Subgraph + reorder(nchw)
```

Comment:
Minor: these comments explain why expected node count was `10` (or another number) before. Could you please remove them from there?